### ISSUE TITLE
Create New Contact from Show Group Members has hidden nav bar buttons

### ISSUE THREAD

**Issue by finder39** [signalapp/Signal-iOS#1805]
- [x] I have searched open and closed issues for duplicates

----------------------------------------

### Bug description
When you have a group chat with an unknown number (unknown contact), it opens a `ABUnknownPersonViewController`, from which you can select Create New Contact. This opens a New Contact view controller, which is Apple build and controlled.

### Steps to reproduce
- Be in a group chat with a number for which you do not have a contact (or delete the contact after being in the group)
- Tap on the group name
- Tap List Group Members
- Tap the group member that is not in your contacts (it shows up as a number instead of a name)
- Tap Create New Contact

**Actual result:** 
New Contact screen with hidden nav buttons
**Expected result:** 
New Contact screen should be shown with Title, Cancel button, and Done button

### Screenshots
![screen shot 2017-02-23 at 1 12 50 am](https://cloud.githubusercontent.com/assets/518673/23252509/ed7c9298-f965-11e6-876f-96fdd3ee3b9d.png)

### Device info

**Device**: iPhone 7
**iOS version**: 10.2.1
**Signal version:** Z.Y2.7.1

**Comment by footiecrazie** [comment]
Debugged on iOS 10.2 simulator and the New Contact screen has both Cancel and Done button which is visible when 'Color Blended Layers' is enabled on the simulator. This is not a Signal code issue, since AddressBookUI is deprecated on iOS 10, the behaviour of ABUnknownPersonViewController may be undefined. This could be solved by using ContactsUI instead of AddressBookUI on iOS 10 and above. 
**Note:** Works perfectly on iOS 9

**Simulator screenshot** 
<img width="320" alt="1805 - colorblendedlayers" src="https://cloud.githubusercontent.com/assets/25761722/23357042/22a5501e-fd02-11e6-9005-8f79866cb3e5.png">

**Comment by finder39** [comment]
I put a fix up, I believe I incorrectly linked to the fix. Let me try again.

**Comment by finder39** [comment]
There we go :)

I agree this isn't Signal's, and that it is Apple's. There is still a fix for it though
